app/controllers/admin/users_controller.rb

```diff
@@ -290,6 +290,7 @@ def trust_level_lock
   def approve
     guardian.ensure_can_approve!(@user)
     @user.approve(current_user)
+    StaffActionLogger.new(current_user).log_user_approve(@user)
```

Comment:
Is this the only spot where we approve an user? If not, it might be better to move the log in the `user.approve`  method.